CMakeLists.txt

```diff
@@ -249,6 +249,10 @@ if(ENABLE_AIRTUNES)
   endif()
 endif()
 
+if(TARGET_WEBOS)
+  list(APPEND DEP_DEFINES "-DTARGET_WEBOS=1")
```

Comment:
This likely doesn't belong here. We should probably create a cmake module for `FindWaylandProtocolsWebos.cmake` That should look for `webos-shell.xml`

Reply:
The problem is that `webos-wayland-extensions` does not provide a pkgconfig file for the procol definitions.

Reply:
you can just use find_path(), pkgconfig is not a requirement.